packages/ui-components/src/display/display.js

```diff
@@ -0,0 +1,111 @@
+// @ts-check
+import { assert, details } from '@agoric/assert';
+import { MathKind } from '@agoric/ertp';
+import '@agoric/ertp/exported';
+
+import { parseAsNat } from './natValue/parseAsNat';
+import { stringifyNat } from './natValue/stringifyNat';
+import { parseAsSet } from './setValue/parseAsSet';
+import { stringifySet } from './setValue/stringifySet';
+
+/**
+ *
+ * @param {string} str - string to parse as a value
+ * @param {AmountMathKind=} mathKind - mathKind of the value
```

Comment:
Kris proposed the [] standard because doc generators like it better
```suggestion
 * @param {AmountMathKind} [mathKind] - mathKind of the value
```

Reply:
sounds good, changed